bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -9,6 +9,7 @@
 #![deny(missing_copy_implementations)]
 #![deny(missing_debug_implementations)]
 
+use bottlecap::tags::lambda::tags::EXTENSION_VERSION;
```

Comment:
fix import

Reply:
yeah we can just put this down into the block below

Reply:
will have to be public as well